src/goodbase/build_checker.py

```diff
@@ -23,25 +23,35 @@ class BuildChecks(BaseModel):
     active_tasks: Set of tasks that need to have run to use the build.
     """
 
-    build_variant_regex: List[str]
-    display_name_regex: List[str]
+    build_variant_regex: Optional[List[str]] = []
+    display_name_regex: Optional[List[str]] = []
     success_threshold: Optional[float] = None
     failure_threshold: Optional[float] = None
     run_threshold: Optional[float] = None
     successful_tasks: Optional[Set[str]] = None
     active_tasks: Optional[Set[str]] = None
 
-    def should_apply(self, build_variant: str, display_name: str) -> bool:
+    def should_apply(self, build_variant: Optional[str], display_name: Optional[str]) -> bool:
         """
         Check if the given build variant should apply to these checks.
 
         :param build_variant: Name of build variant to check.
         :param display_name: Display name of build variant to check.
         :return: True if these checks apply to the given build variant.
         """
-        return any(
-            re.match(bv_regex, build_variant) for bv_regex in self.build_variant_regex
-        ) or any(re.match(dn_regex, display_name) for dn_regex in self.display_name_regex)
+        print("build_variant_regex ", self.build_variant_regex)
+        print("build_variant ", build_variant)
+        print("self.display_name_regex ", self.display_name_regex)
+        print("display_name  ", display_name)
```

Comment:
Are these for debugging? If not, can we format them to make it easier to read for users?

Reply:
oh whoops. Will remove these!